sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -56,6 +56,7 @@
 !> 2024-05-09 | Eric James    | Enable reading of clear-sky downwelling shortwave irradiance
 !> 2024-05-10 | Karina Asmar  | Read omega from model output and calculate HGT for hydrostatic runs
 !> 2024-06-25 | Wen Meng      | Add capability to read fhzero as either an integer or float
+!> 2024-08-26 | Karina Asmar  | Add 6-hourly u/v, speed max wind components at 10m agl
```

Comment:
@KarinaAsmar-NOAA We would like these temporal max 10m wind variables used for any UFS-based applications. Please remove '6-houly' from notes.